Makefile

```diff
@@ -13,38 +14,51 @@ clean: ## remove intermediate files
 	coverage erase
 	rm -rf build
 
-quality: ## run quality checks using tox
-	tox -e quality
-
-piptools:
+piptools: ## install pip-compile and pip-sync.
 	pip install -r requirements/pip-tools.txt
 
-_upgrade: piptools
+upgrade-piptools: piptools ## upgrade pip-tools using pip-tools.
```

Comment:
Will people use this target explicitly? It used to be hidden.

Reply:
The diff is obfuscating the story a bit: `upgrade-piptools` is a completely new target, whereas `_upgrade` was a hack to set `CUSTOM_COMPILE_COMMAND`, which I removed because I learned a better way to set it. It's a coincidence that they have the same prerequisite.

Anyway, I changed `#` to `##` so that `upgrade-piptools` recipe won't be exposed in `make help`.